audformat/core/database.py

```diff
@@ -433,6 +441,385 @@ def duration(file: str) -> pd.Timedelta:
 
         return y
 
+    def get(
+            self,
+            scheme: str,
+            additional_schemes: typing.Union[str, typing.Sequence] = [],
+            *,
+            tables: typing.Union[str, typing.Sequence] = None,
+            splits: typing.Union[str, typing.Sequence] = None,
+            strict: bool = False,
+            original_column_names: bool = False,
+            aggregate_function: typing.Callable[
+                [pd.Series],
+                typing.Any
+            ] = None,
+    ) -> pd.DataFrame:
+        r"""Get labels by scheme.
+
+        Return all labels
+        that match the requested ``scheme``.
+        The request can be limited to selected ``tables``
+        and/or ``splits``
+        and annotated by ``additional_schemes``
+        drawn from the whole database.
+        If ``strict`` is ``False``,
+        a scheme is defined more broadly
+        and does not only match
+        schemes of the database,
+        but also columns with the same name
+        or labels of a scheme
+        with the requested name as key.
+        If at least one returned label belongs to a segmented table,
+        the returned data frame has a segmented index.
+        An ``aggregate_function`` can be provided
+        that specifies how values are combined
+        if more than one value is returned
+        per index entry.
+
+        Args:
+            scheme: scheme ID for which labels should be returned.
+                The search for the labels
+                can be limited
+                by the ``tables`` and ``splits`` arguments
+            additional_schemes: scheme ID or sequence of scheme IDs
+                for which additional labels should be returned.
+                The search is not affected
+                by the ``tables`` and ``splits`` arguments
+            tables: limit search for ``scheme`` to selected tables
+            splits: limit search for ``scheme`` to selected splits
+            strict: if ``False``
+                not only values with the associated scheme(s)
+                are returned,
+                but also values
+                from columns with a column ID matchting the scheme
+                and values from scheme labels,
+                for matching dictionary keys
+            original_column_names: if ``True``
+                return the requested labels
+                under their original column names.
+                For mapped schemes,
+                the column name before mapping is returned,
+                e.g. when requesting ``'gender'``
+                it might return a column named ``'speaker'``
```

Comment:
```suggestion
                keep the original column names
                (possibly results in multiple columns).
                For mapped schemes,
                the column name before mapping is returned,
                e.g. when requesting ``'gender'``
                it might return a column named ``'speaker'``
```